tools/java/org/unicode/cldr/tool/ChartDelta.java

```diff
@@ -69,11 +69,8 @@
     private static final PageId DEBUG_PAGE_ID = PageId.DayPeriod;
```

Comment:
It makes it much longer to read when you do a bunch of mechanical cleanups at the same time, since I have to sift through everything to make sure that it isn't relevant. There has to be some way to communicate that only X is expected to be a material change, and all else is cleanup. Maybe add your own review comments?